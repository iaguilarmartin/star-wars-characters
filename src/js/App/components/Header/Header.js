import React from 'react';
import styled from '@emotion/styled';

import fonts from '../../styles/fonts';
import colors from '../../styles/colors';

const StyledHeader = styled.header`
  text-align: center;
`;

const H1 = styled.h1`
  font-size: ${fonts.sizes.xxl};
  font-family: ${fonts.Starjout};
  font-weight: normal;
  color: ${colors.white};

  small {
    color: ${colors.yellow};
    font-family: ${fonts.Starjhol};
    font-size: ${fonts.sizes.xl};
    display: block;
  }
`;

const Header = () => (
  <StyledHeader>
    <H1>
      Star Wars<small>Characters</small>
    </H1>
  </StyledHeader>
);
export default Header;
