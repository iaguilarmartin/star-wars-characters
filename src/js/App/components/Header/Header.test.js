import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Header from './Header';

describe('<Header/>', () => {
  it('renders', () => {
    shallow(<Header />);
  });

  it('matches the snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
