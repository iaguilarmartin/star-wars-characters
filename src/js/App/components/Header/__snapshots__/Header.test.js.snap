// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> matches the snapshot 1`] = `
<StyledHeader>
  <H1>
    Star Wars
    <small>
      Characters
    </small>
  </H1>
</StyledHeader>
`;
